libcontainerd/container_solaris.go

```diff
@@ -0,0 +1,13 @@
+package libcontainerd
+
+type container struct {
+	containerCommon
+
+	// Platform specific fields are below here. There are none presently on Solaris.
```

Comment:
This doesn't make sence to me (the comment state there are none, but there is some — same comment on `container_windows.go` @jhowardmsft). Also `options` and `ocSpec` doesn't seemed to be used anywhere.